src/log_manager/leader_log_manager.cc

```diff
@@ -53,7 +50,13 @@ void LeaderLogManager::Push(const TermID &term_id,
     all_log_entries_[curr_log_index_] = entry;
 }
 
-void LeaderLogManager::Run() {
+void LeaderLogManager::Run(std::unordered_map<int64_t, LogEntry> &logs,
```

Comment:
```suggestion
void LeaderLogManager::Run(const std::unordered_map<int64_t, LogEntry> &logs,
```

Reply:
for no copy, I use "swap", nonleadermanager and leadermanager 's logs swap, so no nedd const